src/Sarif/ToDotNet/ToDotNet.targets

```diff
@@ -19,7 +19,7 @@
       <FileWrites Include="$(_ObjectModelOutputDirectory)" />
     </ItemGroup>
     
-    <Exec Command="&quot;$(NuGetPath)\NuGet.exe&quot; restore &quot;$(MSBuildThisFileDirectory)packages.config&quot; -PackagesDirectory &quot;$(MSBuildThisFileDirectory)\..\..\packages&quot; -ConfigFile &quot;$(NuGetConfigPath)&quot; -Verbosity quiet" />
+    <!--<Exec Command="&quot;$(NuGetPath)\NuGet.exe&quot; restore &quot;$(MSBuildThisFileDirectory)packages.config&quot; -PackagesDirectory &quot;$(MSBuildThisFileDirectory)\..\..\packages&quot; -ConfigFile &quot;$(NuGetConfigPath)&quot; -Verbosity quiet" />-->
```

Comment:
Add a comment explaining why this is commented out: because the newest version of the JsonSchemaToDotNet code generator, which we need, isn't yet published as a NuGet package, so for now we consume it as a foreign checked-in binary.